CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Accept request payload as java.io.InputStream for Java
```

Comment:
Can we add some note about its limitations and what's not supported?

Reply:
added in https://github.com/mdsol/mauth-jvm-clients/pull/132/commits/ac9b72d9407033a365bbc428dfdd2315ead45826